src/solver/batchnorm/backward_ck.cpp

```diff
@@ -160,7 +160,7 @@ bool BnCKBwdBackward::IsApplicable(
     [[maybe_unused]] const ExecutionContext& context,
     [[maybe_unused]] const miopen::batchnorm::ProblemDescription& bn_problem) const
 {
-#if MIOPEN_BACKEND_HIP || MIOPEN_USE_COMPOSABLEKERNEL
+#if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
```

Comment:
@bghimireamd : please check this line and the two changed files. It seems that #2385 has broken `MIOPEN_USE_COMPOSABLEKERNEL=Off`

Reply:
I think these changes should belong to a separate PR, @bghimireamd @JehandadKhan could you create one?

Reply:
good catch! I will create separate PR for this

Reply:
PR created [#2464 ](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2464)